doc/nrf/gs_assistant.rst

```diff
@@ -7,24 +8,28 @@ Installing automatically
    :local:
    :depth: 2
 
-`nRF Connect for Desktop`_, a cross-platform tool available for Windows, Linux, and macOS, provides different applications that simplify installing the |NCS|.
-Use the :ref:`gs_app_tcm` application to install the |NCS| automatically.
+Complete the steps below to install the |NCS| automatically using the Toolchain Manager application.
+The application installs the full toolchain for the |NCS|, including the |nRFVSC| and the |NCS| source code.
+
+.. rst-class:: numbered-step
+
+Install prerequisites
+*********************
 
 Before you start setting up the toolchain, install available updates for your operating system.
 See :ref:`gs_recommended_versions` for information on the supported operating systems and Zephyr features.
 
-.. _gs_app_tcm:
+Additionally, if you are using a M1-based Mac machine, install the Universal version of SEGGER J-Link.
```

Comment:
Add "This is required for SEGGER J-Link to work correctly on Mac machines". TBC with Simen Tinderholt.

Reply:
Updated as discussed with Simen: no need to restrict this prerequisite to only Mac machines.